IFC4x3/Properties/n/NumberOfPowerSupplyPorts_2zCP6ydR933h5ygkKmdVFt/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfPowerSupplyPorts_2zCP6ydR933h5ygkKmdVFt" Name="NumberOfPowerSupplyPorts" UniqueId="bd3191bc-9db2-430e-b17c-aae5309df3f7" PropertyType="p_tablevalue" PrimaryDataType="IfcLabel" />
```

Comment:
Missed this, if this is just a list like:
|  |
|----|
|Power1|
|Power2|
|etc...|

then the type should be: `PropertyType="p_listvalue"`

If it is suppposed to be:
| No. | label |
|-----|----|
|1|Power1|
|2|Power2|
|3|etc...|

then it should have `PrimaryTye="IfcInteger" SecondaryType="IfcLabel"`